lib/src/layer/tile_layer/tile_bounds/tile_bounds.dart

```diff
@@ -124,30 +123,30 @@ class WrappedTileBounds extends TileBounds {
 
     final tzDouble = zoom.toDouble();
 
-    Tuple2<int, int>? wrapX;
+    (int, int)? wrapX;
     if (crs.wrapLng != null) {
       final wrapXMin =
-          (crs.latLngToPoint(LatLng(0, crs.wrapLng!.item1), tzDouble).x /
+          (crs.latLngToPoint(LatLng(0, crs.wrapLng!.$1), tzDouble).x /
                   _tileSize)
               .floor();
       final wrapXMax =
-          (crs.latLngToPoint(LatLng(0, crs.wrapLng!.item2), tzDouble).x /
+          (crs.latLngToPoint(LatLng(0, crs.wrapLng!.$2), tzDouble).x /
                   _tileSize)
               .ceil();
-      wrapX = Tuple2(wrapXMin, wrapXMax - 1);
+      wrapX = (wrapXMin, wrapXMax - 1);
     }
 
-    Tuple2<int, int>? wrapY;
+    (int, int)? wrapY;
     if (crs.wrapLat != null) {
```

Comment:
Same here, use pattern matching:
```dart
if (crs.wrapLat case final wrapLat?) {
  final wrapYMin = (crs.latLngToPoint(LatLng(wrapLat.$1, 0), tzDouble).y / _tileSize).floor();

  // ...
}
```

Reply:
resolved in https://github.com/fleaflet/flutter_map/pull/1517/commits/3a58ee8354ff119a5168f94eef54aa1ac6a2f373